hack/tools/catalogs/download-catalog

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+SCRIPT_ROOT=$(dirname "$(realpath "$0")")
+source "${SCRIPT_ROOT}/lib/utils.sh"
+
+# Constants
+SERVICE_NAMESPACE="olmv1-system"
+SERVICE_NAME="catalogd-catalogserver"
+LOCAL_PORT=8001
```

Comment:
Can a user override this value easily? I can't remember off the top of my head if `=` sets a variable if the environment variable doesn't already exist in bash

Reply:
I was originally thinking of them as constants. If the values are already set, they will be overriden in the sub-shell the script gets executed in. It shouldn't affect the users environment. However, it could be useful to be able to override them. So, I've made the necessary changes to use the environment value (or default) and also made the variable names a bit less generic.